mods/cnc/rules/misc.yaml

```diff
@@ -22,8 +22,18 @@ CRATE:
 		Condition: cloak-crate-collected
 	GiveMcvCrateAction:
 		SelectionShares: 0
-		NoBaseSelectionShares: 120
+		NoBaseSelectionShares: 50
 		Units: mcv
+	ExplodeCrateAction:
```

Comment:
The Atomic weapon's SpreadDamage warheads need "AffectsParent: true" to fix that.